### PR TITLE
Fixes PaloAltoNetworks/pandevice#72 - fixes and renames bulk operations

### PR DESCRIPTION
**Pull request by shinmog**
Also added new example script, updated previous bulk example script, fixed live tests.

New script works against .25, all live tests pass.

### CODE REVIEW

**Comment by shinmog** [comment]
Note - This PR requires this other PR be merged first, as I'm referencing the new `firewall.Firewall.organize_into_vsys()`:

https://github.com/PaloAltoNetworks/pandevice/pull/79

File: examples/bulk_subinterfaces.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+# WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+# MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+# ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+# WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+# ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+# OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+"""
+bulk_subinterfaces.py
+==========================
```
Comment (btorresgil) [inline examples/bulk_subinterfaces.py]: Heading underline is too long for heading.  This causes warnings during document generation.

File: tests/live/test_network.py [hunk 2]
```diff
@@ -379,11 +379,10 @@ def update_state_obj(self, fw, state):
...
 
     def cleanup_dependencies(self, fw, state):
-        for x in state.management_profiles:
-            try:
-                x.delete()
-            except Exception:
-                pass
+        try:
+            state.management_profiles[0].delete_similar()
+        except Exception:
```
Comment (btorresgil) [inline tests/live/test_network.py]: It's usually bad practice to have `except Exception:`  Can you be more specific about which exception to pass?

**Comment by btorresgil** [comment]
There are a few codacy issues to fix.

**Pull request merged**